### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by ghost** [matterport/Mask_RCNN#2544]
I am training the model with a pretty small training set of 142 images. To mitigate this problem, I used augmentation, however the model seems to be overfitting in just the first few epochs(training with only heads) 
![mrcnn_issue](https://user-images.githubusercontent.com/62421649/116232098-f6d4c800-a759-11eb-8712-798950d52986.png)
, and the val loss doesn't seem to be reducing below 1.6. Does anyone have any suggestions? I have used the learning rates 0.001 and 0.0001. I am attaching the loss plot and the augmentations I have applied. 

augmentation = iaa.Sequential([
    iaa.Fliplr(0.5),
    iaa.Flipud(0.5),
    iaa.OneOf([iaa.Affine(rotate=90),
               iaa.Affine(rotate=180),
               iaa.Affine(rotate=270)])
])

**Comment by TimNagle-McNaughton** [comment]
Reduce your learning rate even further?
